db/seeds.rb

```diff
@@ -0,0 +1,12 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+Greeting.create(text: 'Hello')
```

Comment:
- [ ] @AlejandraKeber Great work on your application, however, you forgot to add a table to be used by active storage for storing different greetings.